test/suite_implementation.js

```diff
@@ -69,7 +69,8 @@ module.exports = function(style, options, _callback) {
                 [];
 
             map.remove();
-            gl.destroy();
+            gl.getExtension('STACKGL_destroy_context').destroy();
```

Comment:
My reading of [the headless-gl changelog](https://github.com/stackgl/headless-gl/blob/1ccb244b006f79637cb1573dbdfddf0a07cd4ca2/CHANGES.md#400) is that `gl.destroy()` is deprecated in favor of `gl.getExtension('STACKGL_destroy_context').destroy()`.

> What's the difference between the before and after here

No functional difference. `gl.getExtension('STACKGL_destroy_context').destroy()` appears to be the preferred API going forward.

> what prompted the change?

In working on this PR, I reread the `headless-gl` docs and changelog to see if I could find any incorrect usage that would explain our test failures or segfaults